src/main/lrsql/admin/interceptors/account.clj

```diff
@@ -12,15 +12,22 @@
 ;; Validation Interceptors
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(def validate-params
+(defn validate-params
   "Validate that the JSON params contain the params `username` and `password`
-   for login and create."
+   for login and delete. If `:strict?` kwarg is true (default) will validate
+   that both meet minimum requirements."
```

Comment:
I think the "minimum requirements" phrasing is vague, since it can also describe "the string just has be non-empty" (which is the case for both strict and non-strict modes). I think it would be better to phrase it as "If `:strict?` kwarg is true (default) then the username and password will need to pass additional requirements."

Reply:
https://github.com/yetanalytics/lrsql/pull/373/commits/e91fd6bbbdb896ff6133c3f7e3511578f4b13d32